rastervision/evaluation/classification_evaluator.py

```diff
@@ -37,4 +39,22 @@ def process(self, scenes, tmp_dir):
                 scene_evaluation = self.create_evaluation()
                 scene_evaluation.compute(ground_truth, predictions)
                 evaluation.merge(scene_evaluation)
+
+            if hasattr(label_source, 'source') and hasattr(
+                    label_source.source, 'vector_source') and hasattr(
+                        label_store, 'vector_output'):
+                tmp_dir = RVConfig.get_tmp_dir().name
+                gt_geojson = label_source.source.vector_source.uri
```

Comment:
If the ground truth label source isn't a `GeoJSONSource`, this won't work. Instead, you need to extract the geojson using `VectorSource.get_geojson()`.

Reply:
Done.